sqlmesh/core/console.py

```diff
@@ -162,13 +176,14 @@ def start_snapshot_progress(
                 BarColumn(bar_width=40),
                 "[progress.percentage]{task.percentage:>3.1f}%",
                 "•",
-                srich.SchedulerBatchColumn(),
+                srich.BatchColumn(),
```

Comment:
what's this change?

Reply:
This class is not related to "Scheduler" in any way, it's just a generic template to create the `x/total` column in rich.